webrtc/src/peer_connection/sdp/mod.rs

```diff
@@ -540,19 +540,44 @@ pub(crate) async fn add_transceiver_sdp(
         );
     }
 
+    // This is equivalent to addSenderSDP in Pion (now)
```

Comment:
Is this comment accurate?

Reply:
Yes, but that was a note for me, and I'll just delete it now the work is done